Core/src/Geometry/ProtoLayer.cpp

```diff
@@ -16,89 +16,59 @@
 using Acts::VectorHelpers::perp;
 using Acts::VectorHelpers::phi;
 
-namespace Acts {
+namespace Acts::detail {
```

Comment:
At some point we need to clarify what we put in details and what we don't. Here it makes a lot of sense though.

Reply:
Fair. In this case, I'm using it to indicate it shouldn't be used directly.